.gitignore

```diff
@@ -1,36 +1,33 @@
-*.gem
-*.rbc
-/.config
-/.idea
-/coverage/
-/InstalledFiles
-/node_modules
-/pkg/
-/spec/reports/
-/spec/examples.txt
-/test/tmp/
-/test/version_tmp/
-/tmp/
+# Gradle files
+.gradle/
+build/
 
-# Used by dotenv library to load environment variables.
-# .env
+# Local configuration file (sdk path, etc)
+local.properties
 
-## Documentation cache and generated files:
-/.yardoc/
-/_yardoc/
-/doc/
-/rdoc/
+# Log/OS Files
+*.log
 
-## Environment normalization:
-/.bundle/
-/vendor/bundle
-/lib/bundler/man/
+# Android Studio generated files and folders
+captures/
+.externalNativeBuild/
+.cxx/
+*.apk
+output.json
 
-# for a library or gem, you might want to ignore these files since the code is
-# intended to run in multiple environments; otherwise, check them in:
-# Gemfile.lock
-# .ruby-version
-# .ruby-gemset
+# IntelliJ
+*.iml
+.idea/
+misc.xml
+deploymentTargetDropDown.xml
+render.experimental.xml
```

Comment:
Same for these, Not sure we need to ignore it 🤔  Correct me if I'm wrong 🙏

Reply:
@Wadeewee they aren't supposed to be tracked by VC.